### ISSUE TITLE
SELECT * plus temporal columns causes renaming

### ISSUE THREAD

**Issue by seancorfield** [xtdb/crux#3085]
```
select p.*, p.xt$system_from, p.xt$system_to from person for all system_time as p
```

This causes the results to contain `xt$system_from:1` and `xt$system_to:1`.

```
select p.*, p.xt$system_from from person for all system_time as p
```

This causes the results to contain `xt$system_from:1`.

```
select p.*, p.xt$valid_from, p.xt$valid_to, p.xt$system_from, p.xt$system_to from person for all system_time as p
```

This produces `xt$system_from`, `xt$system_to:1`, `xt$valid_from`, and `xt$valid_to:1`. 

If `:1` was added consistently with `*`, I'd think "Weird, but I could live with it". However, it doesn't even seem consistent based on that last example.

Expected behavior:

Temporal columns should not get renamed like this.

**Comment by jarohen** [comment]
awaiting a release

**Comment by mbutlerw** [comment]
Hey @seancorfield this should be fixed in the latest `2.0.0-SNAPSHOT`.

Temporal columns are still not included in `*`, however won't be renamed when referred to explicitly. 


We are still addressing some limitations when it comes to duplicate columns, however these should not effect any of your examples from above. 

Do let us know if you run into any further issues, otherwise I'll close this card :smile:

**Comment by seancorfield** [comment]
Confirmed fixed. Feel free to close at this point, unless you have a reason to keep it open.
